source/_posts/2019-03-13-release-89.markdown

```diff
@@ -52,18 +62,18 @@ Experiencing issues introduced by this release? Please report them in our [issue
 
 ## {% linkable_title Breaking Changes %}
 
-- RSSI_PEER and RSSI_DEVICE are different things (fixes #20900) ([@dagobert] - [#20902]) ([homematic docs]) (breaking change)
-- Add yeelight flow action support ([@zewelor] - [#21195]) ([light.yeelight docs]) (breaking change)
-- Prevent partial custom component overlays ([@balloob] - [#21070]) (breaking change)
-- Add support for automatic discovery of TP-Link switches, bulbs and dimmers ([@rytilahti] - [#18091]) ([tplink docs]) ([light.tplink docs]) ([switch.tplink docs]) (breaking change)
-- Enhance SmartThings component subscription ([@andrewsayre] - [#21124]) ([smartthings docs]) (breaking change) (new-feature)
-- Split out iperf3 into a component with a sensor platform ([@rohankapoorcom] - [#21138]) ([iperf3 docs]) ([sensor.iperf3 docs]) (breaking change) (new-platform)
-- Remove the google travel time update service ([@rohankapoorcom] - [#21153]) ([sensor.google_travel_time docs]) (breaking change)
-- Add ADB server functionality to Fire TV ([@JeffLIrion] - [#21221]) ([media_player.firetv docs]) (breaking change)
-- Add SmartThings Cover platform and add cover device classes ([@andrewsayre] - [#21192]) ([cover docs]) ([smartthings docs]) (breaking change) (new-platform)
-- Rewrite of Toon component ([@frenck] - [#21186]) ([toon docs]) (breaking change) (new-platform)
-- Add config for trusted networks auth provider ([@awarecan] - [#21111]) (breaking change)
-- HomeMatic: Add error-attribute ([@dagobert] - [#21009]) ([homematic docs]) (breaking change)
+- RSSI_PEER and RSSI_DEVICE are different things (fixes #20900) - HomeMatic component's wireless actors are having two RSSI values. The way the component was programmed one of them was being overwritten. With this new change, the `rssi` attribute of HomeMatic entities has been removed. As a replacement the attributes rssi_device and rssi_peer have been added. The attribute of `rssi` will no longer be available, you'll need to update to __`rssi_device`__, and you can add `rssi_peer` now as well! ([@dagobert] - [#20902]) ([homematic docs]) (breaking change)
+- Add yeelight flow action support - You can now specifiy an action when calling the yeelight_start_flow service on your yeelights. The __default action will be 'recover'__ if you do not specify something else. This action will be taken after the flow stops.  ([@zewelor] - [#21195]) ([light.yeelight docs]) (breaking change)
+- Prevent partial custom component overlays  Custom platforms that override a built-in platform that has a component, should now also include the component in the custom_components/ folder. So if you want to override `hue/light.py` with a custom version, __you will also need to provide/copy over `hue/__init__.py`.__ ([@balloob] - [#21070]) (breaking change)
```

Comment:
You can drop the commit message in a lot of these cases. So for example for the start of this line, drop "Prevent partial custom component overlays"

Reply:
Also, this is a very noteworthy breaking change 😬 . We should also link for more info to https://developers.home-assistant.io/blog/2019/02/19/the-great-migration.html